eZ/Bundle/EzPublishCoreBundle/Resources/config/storage/legacy/schema.yaml

```diff
@@ -166,6 +166,7 @@ tables:
             is_information_collector: { type: integer, nullable: false, options: { default: '0' } }
             is_required: { type: integer, nullable: false, options: { default: '0' } }
             is_searchable: { type: integer, nullable: false, options: { default: '0' } }
+            is_thumbnail: { type: boolean, nullable: false, options: { default: '0' } }
```

Comment:
Still missing a way to upgrade an app w/o reinstalling db from scratch. If you don't like a concept of upgrade scripts (I see why you wouldn't), then please at least provide that script in our documentation (upgrading from v2 to v3).

Reply:
Fix: https://github.com/ezsystems/ezpublish-kernel/pull/2868/commits/3134e565017bff993793d5dbed4984ca3fc50e77